dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jcache.internal-1.1.feature

```diff
@@ -0,0 +1,19 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.jcache.internal-1.1
+WLP-DisableAllFeatures-OnConflict: false
+visibility=private
+singleton=true
+IBM-ShortName: jcache-1.1
+Subsystem-Version: 1.1
+Subsystem-Name: Java Caching (JCache) 1.1
+-features=\
+  com.ibm.websphere.appserver.classloading-1.0
+-bundles=\
+  io.openliberty.jcache.internal, \
+  com.ibm.websphere.javaee.jcache.1.1
+#
+# TODO Before OLGH 9005 feature GAs, the io.openliberty.jcache.autoapi-1.1 feature should be removed
+#      and the exposure of the javax.cache APIs should be done in this feature.
+#
+kind=ga
```

Comment:
Why is this marked GA?

Reply:
to enable the new function without exposing the new APIs?

Reply:
It needs to be available since the feature is used by other GA features. The functionality is beta gated.